.github/workflows/cri_tests.yml

```diff
@@ -57,3 +57,31 @@ jobs:
     - name: Cleaning
       if: ${{ always() }}
       run: ./scripts/github_runner/clean_cri_runner.sh
+
+  stock-containerd-test:
+    name: Stock Containerd test
+    runs-on: [self-hosted, cri]
+
+    steps:
+    - name: Setup TMPDIR
+      run: mkdir -p $HOME/tmp
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Start Containerd
+      run: sudo containerd &
+
+    - name: Create k8s cluster
+      run: ./scripts/cluster/create_one_node_cluster.sh no-vhive && sleep 2m
+    
+    - name: Create helloworld container
+      run: KUBECONFIG=/etc/kubernetes/admin.conf kn service create helloworld-go --image gcr.io/knative-samples/helloworld-go --env TARGET="vHive CRI test"
+
+    - name: Cleaning
+      if: ${{ always() }}
+      run: ./scripts/github_runner/clean_cri_runner.sh no-vhive
```

Comment:
`no-vhive` -> `stock-only`. here and in the scripts

Reply:
done